Backend/index.js

```diff
@@ -0,0 +1,31 @@
+import "dotenv/config";
+import express from "express";
+import mongoose from "mongoose";
+import cors from "cors";
+
+import resumeRoutes from "./routes/api/v1/resume/index.js";
+
+const app = express();
+
+const PORT = process.env.PORT || 3030;
+
+app.use(cors({ origin: process.env.CORS_ORIGIN }));
```

Comment:
Add localhost url of frontend in `origin` list of cors.
It will help in development environment to access api from localhost url.
It will reduce the work of changing environment variable in `.env` file while in development mode.

Reply:
Fixed in commit d4dea4f